examples/regexp-type-langlib-functions/regexp_langlib_functions.bal

```diff
@@ -0,0 +1,39 @@
+import ballerina/lang.regexp;
+import ballerina/io;
+
+public function main() returns error? {
+
+    string data = "bob@example.net,alice@example.com,charlie@example.org,david@example.xyz,emma@example.io";
+
+    // Split the comma separated email list
+    string[] emails = re `,`.split(data);
+
+    // Print the list of emails
```

Comment:
print is a basic function in Ballerina, So we don't need this comment IMO, wdyt?